tests/integration/helpers.py

```diff
@@ -18,6 +18,19 @@
 APP_NAME = METADATA["name"]
 
 
+def application_active(ops_test: OpsTest, expected_units: int) -> bool:
+    units = ops_test.model.applications[APP_NAME].units
+
+    if len(units) != expected_units:
+        return False
+
+    for unit in units:
+        if unit.workload_status != "active":
+            return False
```

Comment:
**question:** After calling `block_until(len(units) == expected_units)` there is almost always a `wait_for_idle`. Is this section here needed? Does it help to check twice?